### PR TITLE
srcset / blurhash fixes

### PR DESCRIPTION
**Pull request by reekitconcept**

### CODE REVIEW

File: src/components/ImageLoader/makeSrcSet.jsx [hunk 1]
```diff
@@ -65,16 +73,27 @@ const makeSrcSet = (options) => {
...
-          // If no scale available, silently ignore creating a source.
           if (scales.hasOwnProperty(defaultScale)) {
             result.src = createScaledSrc(
               src,
               defaultScale,
               scales[defaultScale],
             ).url;
+          } else {
+            console.log(createMissingScaleSrc);
+            // If no scale is available, use createDefaultSrc instead.
```
Comment (davisagli) [inline src/components/ImageLoader/makeSrcSet.jsx:84]:
```suggestion
            // If no scale is available, use createMissingScaleSrc instead.
```

**Review by davisagli — APPROVED** [review]

**Comment by davisagli** [comment]
Confirmed this fixes the jumping issue I was seeing

**Comment by davisagli** [comment]
@reebalazs but, the builds are not green yet

**Comment by reekitconcept** [comment]
Thank you! - I've fixed the tests.

I don't remember out of my head the solution for the eslint issue. I thought this was solved already. Let's not merge until we figure it out.

**Comment by reekitconcept** [comment]
@davisagli green

**Pull request merged**